pensjon-brevbaker/src/test/kotlin/no/nav/pensjon/brev/template/GenererAlleMaleneTest.kt

```diff
@@ -0,0 +1,72 @@
+package no.nav.pensjon.brev.template
+
+import no.nav.pensjon.brev.Fixtures
+import no.nav.pensjon.brev.TestTags
+import no.nav.pensjon.brev.api.model.maler.Brevkode
+import no.nav.pensjon.brev.maler.ProductionTemplates
+import no.nav.pensjon.brev.renderTestHtml
+import no.nav.pensjon.brev.renderTestPDF
+import org.junit.jupiter.api.Tag
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
+import org.junit.jupiter.params.provider.MethodSource
+
+class GenererAlleMaleneTest {
```

Comment:
Vi har allerede en test som generer pdf for alle autobrev malene, TemplateResourceITest.

Reply:
Du kan bare slette den gamle for min del.